src/actions/index.js

```diff
@@ -0,0 +1,13 @@
+export const createBook = book => ({
+  type: 'CREATE_BOOK',
+  id: book.id,
+  title: book.title,
+  category: book.category,
+});
+
+export const removeBook = book => ({
```

Comment:
- [ ] **[OPTIONAL]** Do we really all book attributes to delete a book, maybe book id is sufficient?